src/liquid/components/ld-button/ld-button.css

```diff
@@ -66,14 +66,12 @@
       );
       color: var(--ld-button-text-color-focus, var(--ld-thm-primary-active));
     }
-    @media (hover: hover) {
-      &:where(:hover) {
-        background-color: var(
-          --ld-button-bg-color-hover,
-          var(--ld-thm-primary-hover)
-        );
-        color: var(--ld-button-text-color-hover, var(--ld-col-wht));
-      }
+    &:where(:hover) {
```

Comment:
That's a good catch indeed. I was experimenting with this in order to find out whether the hover tests were failing because of the media query. It was not the case, but I eventually forgot to revert these changes. I just did and the tests are still fine.